Gemfile

```diff
@@ -21,22 +20,22 @@ gem 'pg', '~> 0.21.0'
 # OFN-maintained and patched version of Spree v2.0.4. See
 # https://github.com/openfoodfoundation/openfoodnetwork/wiki/Tech-Doc:-OFN's-Spree-fork%F0%9F%8D%B4
 # for details.
-gem 'spree_core', github: 'openfoodfoundation/spree', branch: '2-0-4-stable'
+gem 'spree_core', github: 'openfoodfoundation/spree', branch: '2-1-0-stable'
 
 gem 'spree_i18n', github: 'spree/spree_i18n', branch: '1-3-stable'
 
 # Our branch contains two changes
 # - Pass customer email and phone number to PayPal (merged to upstream master)
 # - Change type of password from string to password to hide it in the form
-gem 'spree_paypal_express', github: "openfoodfoundation/better_spree_paypal_express", branch: "2-0-stable"
+gem 'spree_paypal_express', github: 'openfoodfoundation/better_spree_paypal_express', branch: '2-1-0-stable'
 gem 'stripe'
 
 # We need at least this version to have Digicert's root certificate
 # which is needed for Pin Payments (and possibly others).
-gem 'activemerchant', '~> 1.78'
+gem 'activemerchant', '~> 1.78.0'
 
-gem 'devise', '~> 2.2.5'
-gem 'devise-encryptable', '0.2.0'
+gem 'devise', '~> 3.0.1'
+gem 'devise-encryptable'
```

Comment:
isn't there a dependency on the actual devise version? we protect ourselves from backward-incompatible changes with the Gemfile.lock but might be good to point that out in the Gemfile too.

Reply:
now I see we are on 0.2.0 in master and on 0.1.2 in this branch...

Reply:
I will switch to 020 which is the latest and keeep Gemfile as is because 0.2.0 is the latest.